### ISSUE TITLE
Are you still working on this?

### ISSUE THREAD

**Issue by DNDEBUG** [Quaint-Studios/Reia#7]
Title

**Comment by Makosai** [comment]
Yes. I'll actually be resuming development for it in September! I'll be pulling in the netcode and doing some more active open-testing.

**Comment by DNDEBUG** [comment]
i Want to get into something, need a 3D modeller?

**Comment by Makosai** [comment]
Oh? That'd be great! If you have Discord, here's a one-time link you can use to join the Discord for this project.

https://discord.gg/ZyttJXA6

If you don't have Discord, let me know and we'll find a way to get you in on the collaboration.

**Comment by DNDEBUG** [comment]
I Don't have discord

**Comment by Makosai** [comment]
Would you be willing to start using it?

**Comment by DNDEBUG** [comment]
i tried, but it is asking for a phone number
and i don't have one

**Comment by Makosai** [comment]
Did you sign up using a VPN? That, or signing up in certain countries, seems to causing it to ask you to verify your account. The other alternative is Slack. But there's hardly any info for this project there. Discord is the primary location for all of the designs.

**Comment by Makosai** [comment]
Another thing to check is your email for an actual verification email. If you don't see one, check your spam email. I'll work with you to get you in the group.

**Comment by DNDEBUG** [comment]
i did, even after the email verification it still asked for my phone number

**Comment by Makosai** [comment]
And is that via the web browser or via the app?

**Comment by DNDEBUG** [comment]
both

**Comment by Makosai** [comment]
Okay, let's try slack then and I'll try and get you configured in Discord in the future.

https://join.slack.com/t/reiaworld/shared_invite/zt-1dd2woxx2-0gfQYWaQPAnHq3nGYDhTvg

**Comment by Makosai** [comment]
Okay, let's try slack then and I'll try and get you configured in Discord in the future.

https://join.slack.com/t/reiaworld/shared_invite/zt-1dd2woxx2-0gfQYWaQPAnHq3nGYDhTvg
